gaia-fast-list.js

```diff
@@ -1196,7 +1115,10 @@ Internal.prototype = {
       }
     }
 
+    // Can't be more than the last index
+    index = Math.min(index, lastIndex);
```

Comment:
This stung me when the list is shorter that the viewport. Luckily tests caught it.